### DISCUSSION TITLE
Add Left padding on Display Templates

### DISCUSSION THREAD

**Discussion by jamescammarano** [directus/directus#10954] [Feature Requests]
On a display template, it would be nice to be able to control the left padding of numeric fields such as ID and sort. This would be especially useful when working with datasets > 100.

EX of the problem:
<img width="38" alt="Screen Shot 2022-01-10 at 9 58 50 AM" src="https://user-images.githubusercontent.com/67079013/148789402-5e8c237b-0105-4110-84bf-a0565823c590.png">

**Comment by rijkvanzanten** [comment]
Also also also please

```css
font-variant-numeric: tabular-nums;
```

**Reply by benhaynes** [reply]
Iiiintersting... it takes a sans-serif and makes it monospace??

**Reply by rijkvanzanten** [reply]
More or less!

**Reply by benhaynes** [reply]
For the padding, we need to make sure any margin/padding only goes _between_ display elements, not left/right. I'm wondering if we can just (smartly) add this and not have it be an option.

**Reply by rijkvanzanten** [reply]
@benhaynes I'm pretty sure @jaycammarano means "left pad" as in add `0` in front of the value, not actual CSS padding. That way you get

```
008
009
010
011
```

rather than

```
8
9
10
11
```

**Reply by benhaynes** [reply]
Ahhh, "pad with zeros", yes... that's a nice one to add. Not sure how it would work with the `--` NULL values though. We should consider if they can control the character to use when padding, and if we manually or automatically decide how many places to pad (based on all column values or the visible dataset).

**Reply by benhaynes** [reply]
Also, this could partially be resolved by adding alignment options to the Table Layout.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
